x-pack/plugins/uptime/public/components/monitor/ml/__tests__/__snapshots__/ml_manage_job.test.tsx.snap

```diff
@@ -9,15 +9,16 @@ exports[`Manage ML Job renders without errors 1`] = `
   >
     <button
       aria-label="You can enable anomaly detection job or if job is already there you can manage the job or alert."
-      class="euiButton euiButton--primary euiButton--small"
+      class="euiButton euiButton--primary euiButton--small euiButton-isDisabled"
```

Comment:
Don't understand this part, why is it adding disabled state by default?

Reply:
An aspect of the `EuiButton` update I forgot to note: buttons in a loading state (`isLoading` prop), now render a `disabled` button to prevent clicks.